src/components/Nav.js

```diff
@@ -1,20 +1,26 @@
-import { Link } from 'react-router-dom';
-import logo from '../assets/spacelogo.png';
-import './Nav.css';
+import { NavLink } from 'react-router-dom';
+import './css/Nav.css';
 
-const Nav = () => (
-  <nav className="header">
-    <span className="nav-logo">
-      <img src={logo} className="logo" alt="logo" />
-      <span className="logo-title">Space Travellers&apos; Hub</span>
-    </span>
+const Nav = () => {
+  const navs = [
+    { id: 1, path: '/rockets', text: 'Rockets' },
+    { id: 2, path: '/dragons', text: 'Dragons' },
+    { id: 1, path: 'missions', text: 'Missions' },
+    { id: 1, path: '/', text: 'My Profile' },
+  ];
```

Comment:
@emmiiorji  Great job 👏🏽 ,  In the case of id's in the navs array of objects, would like to suggest having unique values as the common best practice, this would separate the styled element uniquely.